certbot-ci/certbot_integration_tests/utils/acme_server.py

```diff
@@ -134,6 +134,9 @@ def _prepare_pebble_server(self):
             [challtestsrv_path, '-management', ':{0}'.format(CHALLTESTSRV_PORT), '-defaultIPv6', '""',
              '-defaultIPv4', '127.0.0.1', '-http01', '""', '-tlsalpn01', '""', '-https01', '""'])
 
+        from certbot_integration_tests.utils import pebble_ocsp_server
```

Comment:
I think we should add a brief comment explaining why this module is imported here.

Reply:
Done.